### PR TITLE
#1007 TestShabondiK8S doesn't check the existence of input argument

### PR DESCRIPTION
**Pull request by chia7712**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #1007)
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-it/src/test/scala/com/island/ohara/it/shabondi/TestShabondiK8S.scala [hunk 1]
```diff
@@ -36,8 +38,7 @@ class TestShabondiK8S extends IntegrationTest with Matchers with Inside {
   assert(sys.env.contains(K8S_API_SERVER_URL_KEY), s"\nCannot find sys.env: $K8S_API_SERVER_URL_KEY")
   assert(sys.env.contains(K8S_API_NODE_NAME_KEY), s"\nCannot find sys.env: $K8S_API_NODE_NAME_KEY")
```
Comment (vitojeng) [inline ohara-it/src/test/scala/com/island/ohara/it/shabondi/TestShabondiK8S.scala]: There should remove the assertion if we need checking the env value first.

**Comment by chia7712** [comment]
I file a small QA for this PR. https://builds.is-land.com.tw/job/PreTest-OHARA/382/console

**Comment by vitojeng** [comment]
> I file a small QA for this PR. https://builds.is-land.com.tw/job/PreTest-OHARA/382/console

This link was broken.

**Comment by chia7712** [comment]
> This link was broken.

The link points to jenkins's public door so you have to switch your network from internal to public...

**Comment by vitojeng** [comment]
> The link points to jenkins's public door so you have to switch your network from internal to public...

Strange...I can't connect at the time, it works now.

**Pull request merged**